docs/zilevm/user-wallets.md

```diff
@@ -0,0 +1,38 @@
+---
+id: user-wallets
+title: Wallets
+keywords:
+  - Wallets
+description: Wallets that can interact with ZILEVM
+---
+
+---
+
+## MetaMask
+
+### Setup
+
+
+:::warning
```

Comment:
Please note that this does not work with MkDocs. It is specific to Docusaurus. Instead use 

```
!!! warning

    Your seed phrase must be backed up and kept secret.
    Funds can be at risk if this phrase cannot be recalled or exposed publically.
    Your Zilliqa seed phrase and your EVM seed phrases are different! Your existing Scilla keys WILL NOT resolve to the same EVM address. DO NOT use scilla's base 16 '0x' address as the target of sending EVM funds to - you will lose your funds!.
```